lib/memcached.rb

```diff
@@ -1,14 +1,7 @@
-require 'memcached_rb'
-
 require 'memcached/client'
```

Comment:
dont you need to declare the require client as last? So it will load the connection and server classes before?